Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Invoke with docker run -p 8000:80 <dockerimageid>
```

Comment:
Is this Docker file needed? This also seems very specific to some operations you may be doing in your own environment.